uber/site_sections/registration.py

```diff
@@ -160,9 +160,7 @@ def delete(self, session, id, return_to='index?'):
                 session.add(Attendee(**{attr: getattr(attendee, attr) for attr in [
                     'group', 'registered', 'badge_type', 'badge_num', 'paid', 'amount_paid', 'amount_extra'
                 ]}))
```

Comment:
See the three lines above this one?  They're doing a better version of what this line below is doing, which is adding a new unassigned badge to replace the assigned badge we're deleting.

Because both of these got left in somehow (probably due to a bad merge on my part with the anthrocon branch or something silly like that) we were adding 2 new badges every time we unassigned 1.  Whoops!